test/hotspot/jtreg/vmTestbase/nsk/jdi/ThreadReference/stop/stop001.java

```diff
@@ -256,16 +268,24 @@ private int runThis (String argv[], PrintStream out) {
                 log2("......getting result from mainThread:");
                 line = pipe.readln();
                 log2("       returned string is: " + line);
-                if (line.equals("null")) {
-                    log3("ERROR: 'stop001a.tObj = e1;' was not assigned");
-                    expresult = returnCode1;
-                } else if (line.equals("equal")) {
-                } else if (line.equals("NOT_equal")) {
-                    log3("ERROR: in the debugee, e1 is not 'LineUnavailableException'");
-                    expresult = returnCode1;
+                if (vthreadMode) {
+                    // Because the stop() called failed, stop001a.tObj should be "null".
```

Comment:
Nit: Is it a typo: `top() called failed` => `stop() call failed` ?

Reply:
Fixed.